content/documentation/tutorials/examples/async-sub.go

```diff
@@ -13,13 +14,24 @@ import (
 func main() {
 
 	// Create server connection: auth and no auth
-	// natsConnection, _ := nats.Connect("nats://foo:bar@localhost:4222")
```

Comment:
I think you may want to leave this comment, this I think to show how one would pass username/password. If you put it back, include `err`